test/test.js

```diff
@@ -259,10 +269,7 @@ describe("Test Screenshots", function() {
       });
     }).then((shotUrl) => {
       assert(shotUrl.startsWith(backend), `Got url ${shotUrl} that doesn't start with ${backend}`);
```

Comment:
Probably makes sense to move this part of the check inside the `verifyShotUrl` method, too, since it's duplicated in both places, and should be easy to compare the `URL` for expected vs actual backend.